capellambse_context_diagrams/context.py

```diff
@@ -860,6 +895,84 @@ def name(self) -> str:  # type: ignore
         return f"Cable Tree View of {self.target.name}"
 
 
+class CustomDiagram(ContextDiagram):
+    """An automatically generated CustomDiagram Diagram."""
+
+    _collect: dict[str, t.Any]
+    _display_symbols_as_boxes: bool
+    _display_parent_relation: bool
+    _hide_direct_children: bool
+    _slim_center_box: bool
+    _display_port_labels: bool
+    _port_label_position: str
+    _transparent_background: bool
+    _display_unused_ports: bool
+    _unify_edge_direction: str
+
+    def __init__(
+        self,
+        class_: str,
+        obj: m.ModelElement,
+        *,
+        render_styles: dict[str, styling.Styler] | None = None,
+        default_render_parameters: dict[str, t.Any],
+    ) -> None:
+        default_render_parameters = {
+            "collect": {},
+            "display_symbols_as_boxes": False,
+            "display_parent_relation": False,
+            "hide_direct_children": False,
+            "slim_center_box": False,
+            "display_port_labels": False,
+            "port_label_position": _elkjs.PORT_LABEL_POSITION.OUTSIDE.name,
+            "transparent_background": False,
+            "display_unused_ports": False,
+            "unify_edge_direction": str,
+        } | default_render_parameters
+        super().__init__(
+            class_,
+            obj,
+            render_styles=render_styles,
+            default_render_parameters=default_render_parameters,
+        )
+        self.collector = custom.collector
+
+
+class PhysicalPortContextDiagram(ContextDiagram):
```

Comment:
We are using the `collect` and `unify_edge_direction` render params which are available for `CustomDiagram`. So it should be:
```suggestion
class PhysicalPortContextDiagram(CustomDiagram):
```